sources/GeneratorSdk/samples/DiaSdk/Microsoft.Dia.Win32Metadata.nuspec

```diff
@@ -17,7 +17,7 @@
     <icon>images\windows.png</icon>
     <repository type="git" url="https://github.com/microsoft/win32metadata.git" />
     <dependencies>
-      <dependency id="Microsoft.Windows.SDK.Win32Metadata" version="49.0.15-preview-g6e729ece55" />
+      <dependency id="Microsoft.Windows.SDK.Win32Metadata" version="53.0.6-preview-g26ed8da0f6" />
```

Comment:
Oh this keeps automatically trickling in every time... How can we get rid of this changing in the background?

Reply:
`.\DoAll.ps1` has `-ExcludePackages -ExcludeSamples` parameters that you may find handy.

Reply:
Ah, [`CONTRIBUTING.md` was recently updated](https://github.com/microsoft/win32metadata/commit/18f30abbe5ad742a86115f0a9aed823fe14280be) with that hint, thanks!